### DISCUSSION TITLE
Lyricsync feedback hub

### DISCUSSION THREAD

**Discussion by DBKarman** [mimuless/lyricsync#2] [Announcements]
Welcome to Lyricsync feedback hub,

Please use this to suggest any changes or report bugs,
I'll try to stay as responsive and update-ive as possible

DBKarman :)

**Comment by gitkps** [comment]
Hello Karman,

Thank you very much for this excellent utility. I tried few other LRC file generators and I wasn't happy. I kept looking for alternatives. Luckily, I landed on your page. Yours is simple, does the job neatly, both on-line and off-line, and portable too. 

I had to fumble a bit initially but soon I figured out. 

Regards and Love from India.
KPS

**Reply by DBKarman** [reply]
Hi,

Thank you so much for the kind response, This was my first project ever, I wrote it as a personal tool because i love music so much but then decided to make it open source and upload it onto github, thats why its all buggy and broken, it went from something with no UI or any thought process to a jumbled up mess molded into a "working thing" and written in a very simple and outdated model of Javascript and the very very old and outdated framework jQuery. You can tell how messy it is by opening the .js file haha. You've definately inspired me to fix it if i can find time the time with university and a job and study :P

Have a nice day :)
DBK

**Comment by ghost** [comment]
Hello! I love this tool! Although, I'd like to ask if you can make a Spotify lyrics generator as an option. As not all songs on Spotify have lyrics and I use them when I play music on discord or in VR. It's rather hard to convert IRC to spotify format.

**Reply by DBKarman** [reply]
Hi! Could you please explain in more details what you require? it seems spotify pulls its lyrics from musixmatch, which you need to be a verified artist on to add lyrics to. and even then you can only add lyrics to your own music.
